apps/i18n/netsim/en_us.json

```diff
@@ -18,6 +18,16 @@
   "hexadecimal": "Hexadecimal",
   "infinity": "Infinity",
   "instructions": "Instructions",
+  "instructions_AddressingMessages": "In this version of the simulator, you will connect to a Router instead of directly to another student. Enter an address in the \"To\" box to send a message to another student.",
+  "instructions_AutomaticDNS": "In this version of the simulator a DNS service is provided for you. Send a message in the form GET [hostname] to the DNS service to get a student's address.",
+  "instructions_default": "You have loaded NetSim in superuser mode.",
+  "instructions_SendAB": "UNDER CONSTRUCTION: This version of NetSim is not yet functional. It will allow students to manually set/read individual As and Bs on a simplex wire.",
+  "instructions_SendAscii": "UNDER CONSTRUCTION: This version of NetSim is not yet functional. It will allow students to send ASCII to each other.",
+  "instructions_SendBits": "UNDER CONSTRUCTION: This version of NetSim is not yet functional. It will allow students to send binary as zeroes and ones to one another",
```

Comment:
Hm... maybe.  I feel like I also need the "Under Construction" notes for the sake of helping Brooke and Baker know which versions of NetSim are usable right now and which ones aren't; they've been encouraging me to "outline" the progression, but that means some non-functional level stubs right now.